data/os/Windows/win116421h2.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
We can delete this file. It will not be in within the hiera lookup.

Reply:
Sure, removing it!